include/seqan3/core/concept/tuple.hpp

```diff
@@ -76,11 +76,14 @@ concept tuple_get_concept = requires (tuple_t & v, tuple_t const & v_c)
     requires std::tuple_size_v<tuple_t> > 0;
 
     typename std::tuple_element<0, tuple_t>::type;
-    { std::get<0>(v)              } -> typename std::tuple_element<0, tuple_t>::type &;
-    { std::get<0>(v_c)            } -> typename std::tuple_element<0, tuple_t>::type const &;
-    { std::get<0>(std::move(v))   } -> typename std::tuple_element<0, tuple_t>::type &&;
+    { get<0>(v)              } -> typename std::tuple_element<0, tuple_t>::type;
```

Comment:
Could we add `namespace stl; get<0>(v)` for adl support?

Reply:
get without namespace already does ADL. You cannot include namespaces inside constraint declarations AFAIK.